dll/win32/shell32/systray.cpp

```diff
@@ -1,207 +0,0 @@
-/*
```

Comment:
Why delete this file?
Can just keep the Shell_NotifyIcon[A|W] in here?
Now you destroy the history of those functions.

Reply:
It is, but not if you edit more than x%.

Reply:
Indeed. The file is now detected as "deleted" and the CUserNotification.cpp as "added".

Reply:
To clarify: Git doesn't store renames at all. What it does is simply that when you ask for a file's history, and the file was created in a certain commit, if the "follow renames" option is enabled, it will look for a deleted file **in the same commit**, that has more than a certain % of the contents identical.

In the low level, git works with "blobs" and it doesn't care what filename they have -- when computing differences (for compression), it finds some other random blob that has similar contents, and this other blob doesn't need to be related in any way, just be a decent match for compression purposes.

Reply:
Fixed.